adapters/ion-adapter/src/test/java/com/connexta/replication/adapters/ion/IonNodeAdapterTest.java

```diff
@@ -80,22 +80,33 @@ public void createResource() {
   }
 
   @Test
-  public void createResourceFailed401() {
+  public void createResource200() {
```

Comment:
✏️ Should probably be adding another test with another 2xx code to properly verify your change above.

Reply:
There is a test above this where we respond with a 202. Or did you mean 2 2xx tests isn't enough?

Reply:
I missed it.